src/Payum/Stripe/Action/AuthorizedAction.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Payum\Stripe\Action;
+
+use Payum\Core\Action\PaymentAwareAction;
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\Request\Authorize;
+use Payum\Core\Request\Sync;
+use Payum\Stripe\Request\Api\CreateCharge;
+use Payum\Stripe\Request\Api\ObtainToken;
+
+class AuthorizeAction extends PaymentAwareAction
```

Comment:
`src/Payum/Stripe/Action/AuthorizedAction.php` should be
`src/Payum/Stripe/Action/AuthorizeAction.php` (for clarity)